frontend/Dockerfile

```diff
@@ -8,6 +8,9 @@ WORKDIR /app
 COPY package.json package-lock.json ./
 RUN npm install
 
+# Legacy imports are installed separately
+RUN npm install @codemirror/legacy-modes
+
```

Comment:
Why not put this in `package.json` tho?
Since ur code seems to use it, and the ppl working on frontend will need to also install it via `npm install` (ie. install dependencies in `package.json`)

Reply:
done